main_concept/indexes.md

```diff
@@ -83,29 +81,29 @@ The fields are [sorted by relevancy importance](/main_concept/indexes.md#fields-
 
 The `id` field is the unique identifier of a movie document. We want to display it (*displayed*) and give the possibility to users to search for the movie using his ID (*indexed*).
 
-The `title` field is an important field, which is why it is ranked second in the order of fields. We want 
+The `title` field is an important field, which is why it is ranked second in the order of fields. We want
 our search queries to search inside this field (*indexed*), and we also want to showcase it to the user (*displayed*).
 
 The `description` field is exactly like the title field. Its 3rd position in the order of the fields puts it after `Title` in relevancy.
 
-The `release_date` field is not indexed because user does not usually search by date. With the *ranked* tag, we can create a custom ranking rule
+The `release_date` field is not indexed because the user does not usually search by date. With the *ranked* tag, we can create a custom ranking rule
```

Comment:
```suggestion
The `release_date` field is not indexed because the user does not usually search for a specific date. With the *ranked* tag, we can create a custom ranking rule
```